libexec/ramalama/ramalama-run-core

```diff
@@ -68,19 +68,11 @@ def main(args):
         with open(os.devnull, 'w') as devnull:
             os.dup2(devnull.fileno(), sys.stderr.fileno())
 
-        args = Namespace(
-            container=False, dryrun=False, engine=None, podman_keep_groups=False,
-            image=None, runtime='llama.cpp',
-            store=os.path.expanduser("~/.local/share/ramalama"), use_model_store=False,
-            quiet=False, debug=False, subcommand='serve', ngl=parsed_args.ngl, threads=parsed_args.threads,
-            temp=parsed_args.temp, authfile=None, env=[], device=None, name=None,
-            oci_runtime=None, privileged=False, pull='newer', seed=None,
-            tlsverify=True, context=parsed_args.context, runtime_args=[], network=None,
-            detach=False, host='127.0.0.1', webui='on', generate=None, port=str(port),
-            rag=None, MODEL=parsed_args.MODEL, func=lambda: None,
-            UNRESOLVED_MODEL=parsed_args.MODEL
-        )
-        serve_cli(args)
+        with open(os.devnull, 'w') as devnull:
```

Comment:
Looks like  you duplicated this code?

Reply:
It's deliberate, one redirects stderr to null, the other redirects stdout to null.

Could create a function for this with a parameter for stderr/stdout seems I just didn't because it seemed like overkill for something that's 2 lines and called twice

Reply:
Ok I missed the subtle difference.

Reply:
Changed it anyway to try and make it more clear